templates/tomcat/server.xml.erb

```diff
@@ -81,11 +81,9 @@
                sslProtocols="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join(",") %>"
                sslEnabledProtocols="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join(",") %>"
                keystoreFile="<%= scope['::candlepin::keystore_file'] %>"
-               truststoreFile="<%= scope['::candlepin::truststore_file'] %>"
```

Comment:
Is it intentional that we no longer support the truststore? It looks like after this it's only used in the broker.

Reply:
We are "falsely" setting a truststore here and not using client authentication through SSL when communicating with Candlepin so having one defined on Tomcat serves as a bit of a misnomer.

Reply:
For reviewing it's easier if such a thing is at least a separate commit. As a user I wouldn't expect this to change when I saw `Fixes #30320: Manage certificate file ownership and permissions` in the changelog. As a developer I wouldn't look for it in the git log if there's a regression around this.

Reply:
This change *is* related to the work in this PR. But I *can* split this change out on it's own.

Reply:
Perhaps it should be a commit `Refs #30320 - drop truststore option from server.xml` as part of the same PR?